modules/spring-config/src/main/resources/minimal-mule-config.xml

```diff
@@ -18,30 +16,57 @@
+        <!--while keeping the local version intact. When the cluster core extension is not active, both-->
+        <!--the default and local beans resolve to the same instance-->
 
-        <spring:bean name="_defaultPersistentObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultPersistentObjectStore"/>
+        <spring:bean name="_defaultInMemoryObjectStore" class="org.mule.config.spring.factories.ConstantFactoryBean">
+            <spring:constructor-arg ref="_localInMemoryObjectStore"/>
+        </spring:bean>
+
+        <spring:bean name="_localInMemoryObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultInMemoryObjectStore"/>
+
+        <spring:bean name="_defaultPersistentObjectStore" class="org.mule.config.spring.factories.ConstantFactoryBean">
+            <spring:constructor-arg ref="_localPersistentObjectStore"/>
+        </spring:bean>
 
-        <spring:alias name="_defaultPersistentObjectStore" alias="_localPersistentObjectStore" />
+        <spring:bean name="_localPersistentObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultPersistentObjectStore"/>
 
-        <spring:bean name="_defaultUserObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultUserObjectStore"/>
+        <spring:bean name="_defaultUserObjectStore" class="org.mule.config.spring.factories.ConstantFactoryBean">
+            <spring:constructor-arg ref="_localUserObjectStore"/>
+        </spring:bean>
 
-        <spring:alias name="_defaultUserObjectStore" alias="_localUserObjectStore" />
+        <spring:bean name="_localUserObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultUserObjectStore"/>
 
-        <spring:bean name="_defaultTransientUserObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultUserTransientObjectStore"/>
+        <spring:bean name="_defaultTransientUserObjectStore" class="org.mule.config.spring.factories.ConstantFactoryBean">
+            <spring:constructor-arg ref="_localTransientUserObjectStore"/>
+        </spring:bean>
 
-        <spring:alias name="_defaultTransientUserObjectStore" alias="_localTransientUserObjectStore" />
+        <spring:bean name="_localTransientUserObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultUserTransientObjectStore"/>
 
-        <spring:bean name="_muleQueueManager" class="org.mule.util.queue.DelegateQueueManager"/>
+        <spring:bean name="_muleQueueManager" class="org.mule.config.spring.factories.ConstantFactoryBean">
+            <spring:constructor-arg ref="_localQueueManager"/>
+        </spring:bean>
 
-        <spring:alias name="_muleQueueManager" alias="_localQueueManager"/>
+        <spring:bean name="_localQueueManager" class="org.mule.util.queue.DelegateQueueManager"/>
```

Comment:
Are you sure that EE is not going to use the local que manager instead of the clusterized one?